infrastructure/lambda/template.yaml

```diff
@@ -583,6 +583,8 @@ Resources:
             TableName: !Ref SessionTable
         - DynamoDBReadPolicy:
             TableName: !Ref PersonIdentityTable
+        - DynamoDBWritePolicy:
+            TableName: !Ref KBVAnswerTable
```

Comment:
I'd prefer the `DynamoDBWritePolicy`'s to be added as part of the next sub-task in this story, which is to do with using the table.

Reply:
I think it makes sense to encapsulate all `template.yaml` changes in this PR. Then the subsequent PR can just be java code amends only